x-pack/plugins/cloud_security_posture/public/common/hooks/use_benchmark_dynamic_values.ts

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { useCspIntegrationLink } from '../navigation/use_csp_integration_link';
+import { CSPM_POLICY_TEMPLATE, KSPM_POLICY_TEMPLATE } from '../../../common/constants';
+import { BenchmarksCisId } from '../../../common/types/benchmarks/v2';
+
+type BenchmarkDynamicNames =
+  | {
+      integrationType: 'CSPM';
+      integrationName: 'AWS';
+      resourceName: 'Accounts';
+    }
+  | {
+      integrationType: 'CSPM';
+      integrationName: 'GCP';
+      resourceName: 'Projects';
+    }
+  | {
+      integrationType: 'CSPM';
+      integrationName: 'Azure';
+      resourceName: 'Subscriptions';
+    }
+  | {
+      integrationType: 'KSPM';
+      integrationName: 'Kubernetes';
+      resourceName: 'Clusters';
+    }
+  | {
+      integrationType: 'KSPM';
+      integrationName: 'EKS';
+      resourceName: 'Clusters';
+    };
+
+export type BenchmarkDynamicValues = BenchmarkDynamicNames & {
+  resourcePlurals: string;
```

Comment:
`resourcePlurals` is vague. I'm not sure what resourcePlurals is referring to.  Another descriptive  suggestion `resourceCountLabel`